docs/manual/manual_ifci_control.rst

```diff
@@ -0,0 +1,61 @@
+
+While this can work with to control a single motor module connected to a single UART port on your flight controller, you can also connect multiple motors to a single UART port for control as a bus. An example of the connection is shown below.
+
+.. figure:: ../_static/manual_images/pulsing_propeller/motor_serial_connection.png
+    :align: center
+    :alt: Bussed Serial Connection
+
+    Bussed Serial Connection
+
+This will work without changing the module IDs, but if telemetry is required you will need to set each module's module ID to a different value. This is done in the :ref:`system control client<sc_message_table>` with the 'module_id' parameter. Each motor in your bus should have different module IDs.
+
+To request telemetry with the packed control message the Telemetry ID Tail Byte (TITB) must be set to the module ID that you'd like to request telemetry from. When this is set, the corresponding motor will immediately reply with a telemetry packet. To get every motor's telemetry, the flight controller must ask each motor for telemetry individually. If you are sending control messages on an aircraft with 4 motors at 400hz then you will only receive telemetry at 100hz per motor. The telemetry data is packed in a standard IQUART message and the data structure is described in :ref:`IFCITelemetryData struct<ifcitelemetrydata_note>`.
+
+Some examples of different aircraft configuration's data packets are shown below:
+
+.. figure:: ../_static/manual_images/pulsing_propeller/motor_layouts_quad.png
+    :align: center
+    :scale: 50%
+    :alt: Example quadcopter data packet
+
+    Example quadcopter data packet
+
+.. figure:: ../_static/manual_images/pulsing_propeller/motor_layouts_coaxcopter.png
+    :align: center
+    :scale: 50%
+    :alt: Example coaxcopter data packet
+
+    Example coaxcopter data packet
+
+.. figure:: ../_static/manual_images/pulsing_propeller/motor_layouts_slide_quad.png
+    :align: center
+    :scale: 50%
+    :alt: Example 6-DOF quadcopter data packet
+
+    Example 6-DOF quadcopter data packet
+
+The order of the control values doesn't matter all that much as long as they match between the flight controller and the settings on the motor. Additionally you could have two motors read the same CVI if they should respond the same as or in reverse of each other.
```

Comment:
The control values' order does not mater so long as they match between the flight controller and the module's settings.